tests/test_codebase/test_mmdet/test_mmdet_utils.py

```diff
@@ -29,6 +30,15 @@ def test_pad_with_value():
     assert np.allclose(padded_x.sum(), x.sum(), rtol=1e-03, atol=1e-05)
 
 
+def test_pad_with_value_if_necessary():
```

Comment:
should test with ort and trt since there are two rewrting functions

Reply:
I think this is not necessary since codecov has covered most cases.